src/backend/joanie/core/models/certifications.py

```diff
@@ -225,6 +253,18 @@ def get_document_context(self, language_code=None):
             # - Otherwise use the first entry of the localized context
             localized_context = list(self.localized_context.values())[0]
 
+        # - Inject the assets
+        for index, organization in enumerate(localized_context["organizations"]):
+            signature = self.images.filter(pk=organization["signature_id"]).first()
```

Comment:
What if `signature_id` or `logo_id` is `None` ?

Reply:
`logo` and `signature` will be None.

Reply:
What about doing something like : 
```
signature = self.images.filter(pk=organization["signature_id"]).first() if organization["signature_id"] else None
```